backend/src/main/java/com/revature/happyfarmersmarket/interceptor/AuthenticationInterceptor.java

```diff
@@ -52,6 +52,22 @@ public boolean preHandle(HttpServletRequest request, HttpServletResponse respons
 
         JwtToken token = new JwtToken(authorizationHeader.substring("Bearer ".length()));
         Optional<Claims> claims = this.jwtService.verifyToken(token);
+
+        // get token as string and extract user details
+        String tokenStr = getTokenFromRequest(request);
+
+        if (token != null && jwtService.validateToken(tokenStr)) {
```

Comment:
Instead of doing this we can instantiate the `UserDetails` object directly from the `claims` object above.

Reply:
Done